syntax = "proto3";

package cafm.identity.policy;

import "identity/user_with_roles.proto";

message PolicyAppService_usersWithAccessRolesRequest {
}

message PolicyAppService_usersWithAccessRolesResponse {
  repeated UserWithRoles user_with_roles_items = 1;
  int32 total_item_count = 2;
}


service PolicyAppService {
  rpc users_with_access_roles (PolicyAppService_usersWithAccessRolesRequest) returns (PolicyAppService_usersWithAccessRolesResponse) {}
}